fastf1/tests/test_laps.py

```diff
@@ -350,6 +350,19 @@ def test_calculated_quali_results(source):
     pd.testing.assert_frame_equal(ergast_results, session.results)
 
 
+@pytest.mark.f1telapi
+def test_calculated_race_results():
+    # This session is chosen as it has a combination of DNF drivers and lapped
+    # drivers who are behind by multiple laps
+    session = fastf1.get_session(2020, "Portugal", 'R')
+    session.load(telemetry=False, weather=False)
+
+    # copy and delete (!) before recalculating
```

Comment:
Probably want to remove the "delete" part here. It's a bit confusing as nothing is deleted here.